src/Makefile.am

```diff
@@ -1,34 +1,11 @@
 sbin_PROGRAMS = ke-recv mmc-format mmc-check
-sbin_SCRIPTS = mmc-mount
-bin_PROGRAMS = ke-recv-test
+sbin_SCRIPTS  = mmc-mount
+bin_PROGRAMS  = ke-recv-test
 
-INCLUDES = \
-	`pkg-config --cflags libosso` \
-	`pkg-config --cflags glib-2.0` \
-	`pkg-config --cflags gconf-2.0` \
-	`pkg-config --cflags dbus-1` \
-	`pkg-config --cflags gtk+-2.0` \
-	`pkg-config --cflags hildon-1` \
-	`pkg-config --cflags libhildonmime` \
-	`pkg-config --cflags gudev-1.0` \
-	`pkg-config --cflags libevdev` \
-	-DLOCALEDIR=\"`pkg-config --variable=localedir osso-af-settings`\"
+AM_CFLAGS = $(DEPS_CFLAGS)
+LDADD     = $(DEPS_LIBS)
 
-ke_recv_LDFLAGS = \
-	`pkg-config --libs libosso` \
-	`pkg-config --libs libhildonmime` \
-	`pkg-config --libs gconf-2.0` \
-	`pkg-config --libs gudev-1.0` \
-	`pkg-config --libs libevdev`
-
-mmc_format_LDFLAGS = \
-	`pkg-config --libs hildon-1`
-
-mmc_check_LDFLAGS = \
-	`pkg-config --libs hildon-1`
-
-ke_recv_test_LDFLAGS = \
-	`pkg-config --libs libosso`
+AM_CPPFLAGS = -DLOCALEDIR=\"`pkg-config --variable=localedir osso-af-settings`\"
```

Comment:
this (calling pkgconfig) belongs to configure.ac

Reply:
This was that way initially.
I agree, it's more logical to have it in configure.ac.